DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: worldfootballR
 Title: Extract and Clean World Football (Soccer) Data
-Version: 0.5.10.1000
+Version: 0.5.2
```

Comment:
shouldn't this be `5.11` or `5.12`? we've already had a `5.2`

Reply:
How embarrassing 😳

Thanks for picking that up!